.conform.yaml

```diff
@@ -8,7 +8,7 @@ policies:
           gitHubOrganization: talos-systems
       spellcheck:
         locale: US
-      maximumOfOneCommit: true
+      maximumOfOneCommit: false
```

Comment:
to make it easy to do backports with multiple not squashed commits

Reply:
Good idea